src/BIP32.php

```diff
@@ -64,6 +64,13 @@ class BIP32
     public static $litecoin_testnet_public = '0436f6e1';
     public static $litecoin_testnet_private = '0436ef7d';
     public static $litecoin_testnet_version = '6f';
+    // Monacoin
+    public static $monacoin_mainnet_public = '';
+    public static $monacoin_mainnet_private = '';
+    public static $monacoin_mainnet_version = '32';
+    public static $monacoin_testnet_public = '';
+    public static $monacoin_testnet_private = '';
+    public static $monacoin_testnet_version = '6f';
```

Comment:
do you know the values for the other vars here?

or you have a BIP32 base58 encoded HD priv/pub key so we can figure them out? they're probably the same as bitcoin mainnet I guess (which result in `xpriv..` and `xpub..` prefixes)